runtime/helpers.js

```diff
@@ -22,6 +22,26 @@ function notEmpty(o) {
     return true;
 }
 
+function createLazyRenderer(handler) {
```

Comment:
I agree that we can probably improve the name here. Maybe `createDeferredRenderer` is better? If you don't like that, maybe you can come up with a better name? When there are circular dependencies we may end up with a partially loaded module that does not yet have it's `renderer(input, out)` method. When we detect that situation we create a wrapper renderer function that on first render tries again to resolve the actual `renderer` function on the provided module. I think `defer` is better because it means "put off (an action or event) to a later time; postpone.". Thoughts?